spec/models/dimensions/edition_spec.rb

```diff
@@ -263,93 +252,4 @@
       expect(-> { create :edition, base_path: 'value', latest: false }).to_not raise_error
     end
   end
-
-  describe '.search' do
```

Comment:
Does the `search` method for the dimension editions model still exist?

Reply:
Same as above, not sure if I get your point here. I have removed it because the method no longer exists in the `Dimensions::Edition`

Reply:
Sorry I missed where you removed them! Makes sense now